lms/models/application_settings.py

```diff
@@ -25,7 +58,10 @@ def set(self, group, key, value):
         :param key: The key in that group
         :param value: The value to set
         """
-        self.data.setdefault(group, {})[key] = value
+        super().setdefault(group, {})[key] = value
+
+    def __repr__(self):
+        return f"ApplicationSettings({super().__repr__()})"
```

Comment:
I like doing this:

```suggestion
        return f"{self.__class__.__name__}({super().__repr__()})"
```

As it behaves nicely when sub-classed.